Application/components/Server/MemberList.tsx

```diff
@@ -12,7 +12,7 @@ const generateHash = (input: string) => {
   return createHash('sha256').update(input).digest('hex');
 };
 
-export function MemberList({isAdmin, chatID}: any) {
+export function MemberList({isAdmin, chatID, isView}: any) {
```

Comment:
Good addition of `isView` to not display the admin tab on the right if we are not looking at a text channel in the present view.